ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/StepperScreen.kt

```diff
@@ -27,7 +27,6 @@ import androidx.wear.compose.material.Text
 
 /**
  * Displays a Stepper, which allows users to make a selection from a range of values.
- * https://developer.android.com/reference/kotlin/androidx/wear/compose/material/package-summary#Stepper(kotlin.Int,kotlin.Function1,androidx.compose.ui.Modifier,kotlin.ranges.IntProgression,kotlin.Function0,kotlin.Function0,androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Color,kotlin.Function1)
```

Comment:
Normally, these are ok to keep over 100, I just wasn't sure on the others but it looks like they were all under. :)